src/data/markdown/translated-guides/en/02 Using k6/00 Tutorial/200 Test for performance.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Test for performance
+excerpt: Write thresholds to evaluate performance criteria, then increase load to see how the system performs.
+---
+
+In the previous section, you made a working script to test the endpoint functionality.
+The next step is to test how this system responds under load.
+This requires using the powerful [`options`](/using-k6/options) object, which configures the parts of the test that don't deal with user behavior.
+
+In this tutorial, learn how to:
+- Use [thresholds](/using-k6/thresholds) to assert for performance criteria
+- Configure load increases through [scenarios](/using-k6/scenarios)
+
+These examples build on the script from the previous section.
+
+## Context: meet service-level objectives
+
+To assess the login endpoint's performance, your team may have defined [service level objectives](https://sre.google/sre-book/service-level-objectives/) (SLOs):
+- 99.9% of requests are successful
+- 99% of requests have a latency of 1000ms or less.
+
+The service must meet these SLOs up to peak traffic of 1 request/second.
+
+After you confirm run the peak traffic test, run another test to determine where the system degrades (this is also often done to discover baselines before the organization has an SLOs).
+
+## Assert for performance with thresholds
+
+To codify the SLOs, add [_thresholds_](/using-k6/thresholds) to test that your system performs to its goal criteria.
+Thresholds are exported in options.
+
+
+```javascript
+export const options = {
+  //define thresholds
+  thresholds: {
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(99)<200'], // 99% of requests should be below 200ms
+  },
+};
+```
+
+Add these thresholds object to your script and run it.
```

Comment:
```suggestion
Add this option object with thresholds to your script and run it.
```

I am not certain how explicit this is and if somebody with less experience will not end up copying it inside the `default` function for example :shrug: